testing/workflows/components/workflows.libsonnet

```diff
@@ -162,7 +162,7 @@
             {
               name: "e2e",
               dag: {
-                tasks: [
+                tasks: std.prune([
```

Comment:
why do you need to prune here?

Reply:
For gke, the argo workflow step will be `{}`. prune removes that entry.

Reply:
I thought we just need to prune here https://github.com/kubeflow/kubeflow/blob/master/testing/workflows/components/workflows.jsonnet#L13?

Reply:
I would prefer explicitly pruning the task list rather than relying on pruning at a higher level. I believe pruning at a higher level is not a good practice because sometimes there are parts of the manifest which require an explicit empty object, eg : https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/cert-manager.libsonnet#L7